### ISSUE TITLE
Feature Request: Allow copy chunk from diff

### ISSUE THREAD

**Issue by cloudbow** [ryu1kn/vscode-partial-diff#95]
I consider this to be one of the most important plugin as of today especially with ai tools like Claude with its artifact feature. When I want to copy code from an artifact its a gret feature. And its also useful with chatgpt as well . Please provide copy chunk functionality which is very essential for the diff tool so that I select part of the content from the ai.

**Comment by robkorv** [comment]
Yeah. Copying and/or saving the diff would be an awesome feature.

**Comment by robkorv** [comment]
https://github.com/ryu1kn/vscode-partial-diff/issues/9 is a similar request.

**Comment by ryu1kn** [comment]
Thanks @cloudbow and @robkorv . Consolidating this into #9 . Feel free to reopen this if #9 is not what you want here.
